src/plugins/PluginSlot.jsx

```diff
@@ -54,15 +60,28 @@ const PluginSlot = forwardRef(({
       // If hidden, don't push to finalChildren
       if (!pluginConfig.hidden) {
         let container;
-        // If default content, render children
+        // If default content, render children (merging any custom defined props from
+        // pluginConfig.content with existing props, if any, on `RenderWidget`).
```

Comment:
```suggestion
        // pluginConfig.content with existing props on `RenderWidget`).
```

Nit: Seems a little extra wordy but could just be me 🤷

Reply:
@MaxFrank13 How's this?:

```
// If default content, render children (merging any custom defined props from
// pluginConfig.content with any existing props on `RenderWidget`).
```

Reply:
Very nice! Thanks